src/aqdata/aq_base.h

```diff
@@ -5,7 +5,7 @@
 #include <map>
 #include <unordered_map>
 #include <string>
-
+#include <iomanip>
```

Comment:
I think adding an extra empty line would be a good idea to separate the blocks of STL and third-party libs. See the example: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes